### ISSUE TITLE
Support underscores?

### ISSUE THREAD

**Issue by gilbert** [Paratron/hookrouter#103]
I spent a while figuring out why my route wasn't working. It turns out the problem was hookrouter didn't fully recognize `/docs/:doc_id` as does not support underscores.

Since most routing libraries support them, would you be in favor of allowing them on hookrouter too?

Thanks for this great, simple library.

**Comment by Paratron** [comment]
I made a note to put that in with v 2.0. In the meantime: how about using camel case? `docId` should work fine.

**Comment by zhaoyao91** [comment]
@Paratron wow, is there a v2 on the road? Can't wait for that. I love this lib with some worry about the long-time-no-updating.

**Comment by gilbert** [comment]
> In the meantime: how about using camel case?

Yup, that's what I ended up doing. Didn't mean to imply it was a blocker. Thanks!
